app/src/main/java/org/simple/clinic/home/overdue/OverdueListAdapter.kt

```diff
@@ -53,6 +84,7 @@ class OverdueListViewHolder(
   private val overdueDays by bindView<TextView>(R.id.overdue_days)
   private val callButton by bindView<ImageButton>(R.id.overdue_patient_call)
   private val separator by bindView<View>(R.id.overdue_separator)
+  private val bottomLayout by bindView<LinearLayout>(R.id.overdue_bottom_layout)
```

Comment:
Instead of `bottomLayout`, can we rename this to `actionsContainer` or something?